STY: make allowed line length 9 longer to 88 from 79 (#24893)
matplotlib/matplotlib: doc/devel/coding_guide.rst

* Does the PR conform with the :ref:`coding_guidelines`?
* Is the :ref:`documentation <pr-documentation>` (docstrings, examples,
  what's new, API changes) updated?

> * Is the change only reflowing code/docstrings for increased line length?
>   Generally, such changes are discouraged when not part of other non-stylistic
>   work because it obscures the git history of functional changes to the code.
>   Reflowing a method or docstring as part of a larger refactor/rewrite is
>   acceptable.

Reviewer: This seemed a little out of left-field - maybe more general is better?  

```suggestion
* Is the change just stylistic?  Generally, such changes are discouraged when not 
  part of other non-stylistic work because it obscures the git history of functional 
  changes to the code. Style changes to a method or docstring as part of a larger 
  refactor/rewrite is acceptable.
```
```suggestion
* Is the change only reflowing code/docstrings for increased line length?
  Generally, such changes are discouraged when not part of other non-stylistic
  work because it obscures the git history of functional changes to the code.
  Reflowing a method or docstring as part of a larger refactor/rewrite is
  acceptable.
```
